package.json

```diff
@@ -78,5 +78,8 @@
       "<rootDir>/packages/enzyme-matchers/src"
     ],
     "verbose": true
+  },
+  "dependencies": {
+    "circular-json-es6": "^2.0.1"
```

Comment:
This needs to be a dependency of packages/enzyme-matchers, not this root package.json. Otherwise it won't get bundled up correctly.